Q3.py

```diff
@@ -0,0 +1,19 @@
+# Build a counter generator
+
+def counter():
+     yield 1
+     yield 2
+     yield 3
+     yield 4
+     yield 5
```

Comment:
instead of writing this yields manually, can you make this function to return the next value upon being called each time?
you can think of using a count variable and a while loop to keep generating new values.

Reply:
Noted this approach as well and implemented it.